lisa/util/logger.py

```diff
@@ -30,6 +31,10 @@ def lines(
                 temp_content.append(f"{key}: {value}")
             content = temp_content
         for line in content:
+            # We can't afford to let ANSI escape codes trash our
+            # stdout stream
+            ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
```

Comment:
It's not for all code path. Below `_log` method is for all. BTW, will it break utf-8?

Reply:
No breaking. If UTF-8 and ANSI escape codes clashed we'd have a much more serious problem in all of computing. Updated to skip empty lines. That's the path exercised by everything I see coming from SSH (and will end exactly equal to runtime/runs).